src/display.rs

```diff
@@ -10,13 +10,13 @@
 #![allow(non_upper_case_globals)]
 
 use libc;
-use std::mem;
```

Comment:
Please squash this commit into the previous two as appropriate.

Reply:
@jdm: Thanks, done.